my-website/docs/competicoes/sacup/rulesnrequirements.md

```diff
@@ -6,6 +6,172 @@ sidebar_label: Rules & Requirements
+* Todos os propelentes devem ser não-tóxicos. 
+* A IREC utiliza utiliza padrões nacionais como  NFPA 1127 e FAA. 
+
+## Avaliação 
+
+A avaliação levará em conta a documentação do projeto, a sessão de posters que acontecerá durante a conferência da SA Cup, a qualidade do design e construção do sistema e por fim a eficiência operacional e performance demonstrada na SA Cup. 
+
+## Equipe 
+
+Equipes da IREC devem consistir de membros matriculados em cursos de graduação, mestrado ou doutorado de uma ou mais instituições. Não há limite para a quantidade de membros. 
+
+## Limite de Submissão 
+
+Cada equipe deve submeter apenas um projeto em apenas uma categoria na IREC. Só é permitida uma equipe por universidade. 
+
+## Payload 
+
+- O Foguete deve carregar um payload de até 8.8 lb.
+- É encorajado que os payloads sejam compostos de experimentos criativos e demonstrações tecnológicas.
+- Os foguetes devem ser modelados para entregar o payload no apogeu alvo e recuperá-los independente de qualquer função passiva ou ativa do payload.
+- Payloads de experimentos científicos e demonstrações tecnológicas serão avaliados pelos representantes da Space Dynamics Laboratory (SDL).
+
+### Localização e Interface do Payload
+
+- Não são especificação a localização, integração e remoção do payload. Porém, os oficiais da competição pesarão o payload independente dos sistemas associados ao veículo de lançamento fundamentais para o voo.
+- Se o design do payload impedir este de ser pesado de maneira independete do veículo de lançamento, isso resultará em uma penalidade.
+
+### Materiais do Payload
+- Payloads não devem contem materiais prejudiciais ou radioativos. Se necessário, deve ser aprovado pelos oficiais da competição e não devem ultrapassar  1 µC de atividade.
+- Payloads não podem contem animais vivos ou vertebrados.
+
+### Forma e Dimensões do Payload 
+-  Boiler Plate Payload ( payload não-funcional, apenas para simular a massa): o payload deverá ter a forma de um  ou mais CubeSats, com o máximo de 3U quando empilhados ( 10cm x 10cm x 30cm).
+- Payload com experimento científico ou demonstração tecnológica: Esses tipos de payload podem ser construídos em qualquer fator de forma, contendo o experimento e a sua estrutura e respeitando as normas previamente descritas. 
+
+É encorajado que as equipes adotem o design CubeSat. Equipes que fizerem isso ganharão pontos de bônus.
+
+## Limitações do Foguete 
+
+- Os veículos de lançamento não devem exceder um impulso total de 40.960 Newton-segundos.
+
+## Monitoramento GPS 
+
+- Todos os veículos de lançamento participantes da Spaceport America Cup devem carregar um Sistema de Posicionamento Global a partir de 2021. 
```

Comment:
Se for traduzir GPS coloca a sigla antes. Assim: ...devem carregar um GPS (Sistema de Posicionamento Global)...